app/src/main/java/ch/epfl/sdp/appart/user/UniversityEmailDatabase.java

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.sdp.appart.user;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * The functionality of this class is to store a database of
+ * accepted and trusted university email providers.
+ */
+public class UniversityEmailDatabase {
+
+    /* add here new trusted university email providers */
+    private String[] universityEmailDatabase = {"epfl.ch", "unil.ch", "unige.ch"};
+
+    private Set<String> universityEmailDatabaseSet = new HashSet<String>(Arrays.asList(universityEmailDatabase));
```

Comment:
```suggestion
    private static final Set<String> universityEmailDatabaseSet = new HashSet<String>(Arrays.asList(universityEmailDatabase));
```

Reply:
> Suggested change 
>     private Set<String> universityEmailDatabaseSet = new HashSet<String>(Arrays.asList(universityEmailDatabase));
>     private static final Set<String> universityEmailDatabaseSet = new HashSet<String>(Arrays.asList(universityEmailDatabase));

Good point! Just updated the fixes you suggester @GrapixLeGrand